### PR TITLE
changes to use a clean membership form that will generate new members…

### PR DESCRIPTION
**Pull request by ucichr**
changes to use a clean membership form that will generate new membership entries in our register and start a workflow to grant access to appropriate spaces.

### CODE REVIEW

File: config/volunteerOpportunities.tsx [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
-        throughout the year.
+        <p>
+          This team is responsible for our public facing technology, including primarily our website. If you are
+          interested in development (DDD Perth is probably a good place...) from our voting platform through to our
+          build process, you would be very welcome to join the team.
+        </p>
+        <p>
+          {' '}
+          Our website and other codebases are available on{' '}
+          <SafeLink href="https://github.com/dddwa" target="?_blank">
```
Comment (amykapernick) [inline config/volunteerOpportunities.tsx:20]: Any particular reason it's `target="?_blank"` instead of just `_blank`?

**Comment by ucichr** [comment]
Copied from a different link? No reason. Can change.

**Comment by amykapernick** [comment]
Approved

**Review by dddperth — APPROVED** [review]

**Comment by dddperth** [comment]
Review passed by @amykapernick

**Pull request merged**